manifests/05-service.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
-    service.beta.openshift.io/serving-cert-secret-name: serving-cert
     capability.openshift.io/name: Console
+    service.beta.openshift.io/serving-cert-secret-name: metrics-serving-cert
```

Comment:
Is this likely to cause an issue on upgrade? Are you going to orphan the old secret?

Reply:
good point 👍 lets rollback the name to avoid any issues